scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/RevioPacBioProcessor.java

```diff
@@ -341,58 +331,54 @@ private boolean isRunComplete(File runDirectory, int smrtCellCount) throws IOExc
   }
 
   /**
-   * Check for presence of Transfer_Test file and grab creation time
+   * Filter runDirectory for Transfer_Test files and grab earliest creation time
    *
    * @param runDirectory which we are currently processing
-   * @param dto PacBioNotificationDto which we will modify @Throws NoSuchElementException if the
-   *     .Transfer_Test files are missing
+   * @return earliest file creation time
+   * @throws NoSuchElementException if the .Transfer_Test files are missing
```

Comment:
This is no longer thrown, and the `@return` should include "or null if no Transfer_Test files are found"

Same for `completionTimeFromTransferDone`.

I'm not sure whether failing to find a start time should cause an error... Can leave as is for now